management/grant.go

```diff
@@ -38,3 +40,12 @@ func (m *GrantManager) List(opts ...ListOption) (g []*Grant, err error) {
 func (m *GrantManager) Delete(id string, opts ...ListOption) error {
 	return m.delete(m.uri("grants", id) + m.q(opts))
 }
+
+// Delete revokes all grants associated with a user-id
+// https://auth0.com/docs/api/management/v2#!/Grants/delete_grants_by_id
+func (m *GrantManager) DeleteByUserID(userID string, opts ...ListOption) error {
+	opts = append(opts, func(urls url.Values) {
+		urls.Add("user_id", userID)
+	})
+	return m.delete(m.uri("grants") + m.q(opts))
+}
```

Comment:
Deleting by user id should already be possible:

```go
Delete("", Parameter("user_id", userID))
```

... at least that was the intention of allowing `opts ...ListOption` as an argument to `Delete`. Usually deletes only take an id.

Would this work for you?

Reply:
Yes! I simply missed it.